.github/workflows/detect_unmerged.yml

```
@@ -0,0 +1,26 @@
+---
+
+name: Detect commits in release branch have not been merged to main.
+
+on:
+  push:
+    branch:
```

actionlint (1.7.12)

[error] unexpected key "branch" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)